tests/static/js/jqunit/jqUnit-binder-commonEventBindings.js

```diff
@@ -0,0 +1,128 @@
+/* globals fluid, jqUnit */
+(function () {
+    "use strict";
+    var gpii = fluid.registerNamespace("gpii");
+
+    // Component to test support for common event bindings
+    fluid.defaults("gpii.tests.binder.commonEventBindings", {
+        gradeNames: ["gpii.tests.binder.base", "gpii.binder.bindMarkupEvents"],
+        bindings: {
+        },
+        selectors: {
+            inputButton: "#input-button",
+            inputKeydown: "#input-button",
+            paragraphClick: ".paragraph-click",
+            multipleEventInputButton: "#input-multiple-events-button"
+        },
+        markupEventBindings: {
+            inputButton: {
+                method: "click",
+                args: ["{that}.handleInputClick"]
```

Comment:
Supply further tests to validate the primitive kinds of "compact" IoC syntax we support which allows for args

Reply:
Thanks for this tip, I didn't know this was possibly actually, and will simplify my usage in the capture tool, PPT, and elsewhere